monitor/grafana/zeebe.json

```diff
@@ -1,4 +1,76 @@
 {
+  "__inputs": [
+    {
+      "name": "DS_PROMETHEUS",
+      "label": "Prometheus",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
+    }
+  ],
+  "__requires": [
```

Comment:
:thinking: Uh, this is new. I guess it's fine? :man_shrugging:

Reply:
Yeah I guess so, at least I haven't added this :D